src/Uno.UI/UI/Xaml/Shapes/Shape.cs

```diff
@@ -208,15 +204,11 @@ public DoubleCollection StrokeDashArray
 		);
 		#endregion
 
-#if LEGACY_SHAPE_MEASURE
+#if __WASM__
```

Comment:
We should be able to remove all of those. The properties should probably be batch updated in the DOM only in the `Arrange`